manual/manual/cmds/spacetime.etex

```diff
@@ -0,0 +1,118 @@
+Spacetime on a per-source-file basis or for a subset of files in a project;
+all files involved in the executable being profiled must be built with the
+Spacetime compiler.
+
+If the {\tt libunwind} library is not available on the system then it will
+not be possible for Spacetime to profile allocations occurring within
+C stubs.  If the {\tt libunwind} library is available but in an unusual
+location then that location may be specified to the {\tt configure} script
+using the {\tt -libunwinddir} option (or alternatively, using separate
+{\tt -libunwindinclude} and {\tt -libunwindlib} options).
+
+OPAM switches will be provided for Spacetime-configured compilers.
+
+Once the appropriate compiler has been selected the program should be
+built as normal (ensuring that all files are built with the Spacetime
+compiler).  For many uses it will not be necessary to change the
+code of the program to use the profiler.
+
+Spacetime-configured compilers run slower and occupy more memory than their
+counterparts.  It is hoped this will be fixed in the future as part of
+improved cross compilation support.
+
+\subsection{Running}
+
+Programs built with Spacetime instrumentation have a dependency on
+the {\tt libunwind} library unless that was unavailable at configure time or
+the {\tt -disable-libunwind} option was specified
+(see section\ \ref{runtimeoverhead}).
+
+Setting the {\tt OCAML\_SPACETIME\_INTERVAL} environment variable to an
+integer representing a number of milliseconds before running a program built
+with Spacetime will cause memory profiling to be in operation when the
+program is started.  The contents of the OCaml heap will be sampled each
+time the number of milliseconds that the program has spent executing since the
+last sample exceeds the given number.  (Note that the time base is combined
+user plus system time---{\em not} wall clock time.  This peculiarity may be
+changed in future.)
+
+The program being profiled must exit normally or be caused to exit using
+the {\tt SIGINT} signal (e.g. by pressing Ctrl+C).  When the program exits
+files will be written in the directory that was the working directory when
+the program was started.  One Spacetime file will be written for each
+process that was involved, indexed by process ID; there will normally only
+be one such.  The Spacetime files may be substantial.  The directory to which
+they are written may be overridden by setting
+the {\tt OCAML\_SPACETIME\_SNAPSHOT\_DIR} environment variable before the
+program is started.
+
+Instead of using the automatic snapshot facility described above it is also
+possible to manually control Spacetime profiling.  (The environment variables
+{\tt OCAML\_SPACETIME\_INTERVAL} and {\tt OCAML\_SPACETIME\_SNAPSHOT\_DIR}
+are then not relevant.)  Full documentation as regards this method of profiling
+is provided in the standard library documentation for the
+{\tt Spacetime} module elsewhere in this manual.
```

Comment:
Would it not be nicer to use a reference to either the `Spacetime` or `c:stdlib` label rather than the vague  "elsewhere in this manual"?